addons/spectator/functions/fnc_compat_zeus.sqf

```diff
@@ -12,8 +12,8 @@
 params ["_display"];
 
 _display displayAddEventHandler ["Unload",{
-    // Only re-open if still a spectator
-    if (GVAR(isSet)) then {
+    // Only re-open if still a spectator (and not remote-controlling)
+    if (GVAR(isSet) && {isNull (GETMVAR(bis_fnc_moduleRemoteControl_unit,objNull))}) then {
```

Comment:
Maybe check `player != ace_player`? That would catch other ways similar to remote control that we bypass with `ace_player`.